frontend/src/components/GoalCards/GoalCard.scss

```diff
@@ -79,7 +79,12 @@ $max-width-date-column: 50%;
 
   .ttahub-goal-card__goal-column__created-on,
   .ttahub-goal-card__goal-column__last-tta,
-  .ttahub-goal-card__goal-column__last-reviewed {
+  .ttahub-goal-card__goal-column__entered-by {
     flex-basis: clamp($min-width-date-column, 10%, $max-width-date-column);
   }
-}
\ No newline at end of file
+}
+
+.ttahub-goal-card__entered-by-tooltip > .usa-tooltip__body,
+.ttahub-goal-card__entered-by-tooltip .smart-hub-tooltip:hover .usa-tooltip__body, .smart-hub-tooltip.show-tooltip .usa-tooltip__body {
```

Comment:
No way to get this to stick without important?

Reply:
Unfortunately couldn't find a specific-enough selector to do this without !important 🤷

Reply:
```css
.ttahub-goal-card__entered-by-tooltip > .usa-tooltip__body,
.ttahub-goal-card__entered-by-tooltip.smart-hub-tooltip:hover .usa-tooltip__body, 
.smart-hub-tooltip.show-tooltip .usa-tooltip__body {
  white-space: nowrap; 
}